content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -16,7 +16,7 @@ This document describes _projected volumes_ in Kubernetes. Familiarity with [vol
 
 ## Introduction
 
-A `projected` volume maps several existing volume sources into the same directory.
+A `projected volume` maps several existing volume sources into the same directory.
```

Comment:
The original text is correct; you set the volume type to `projected`, not `projected volume`.

Reply:
The concept is “projected volumes”; the thing you'd type into a manifest is `projected:` (assuming you're using YAML).

Reply:
The change has been implemented.